Exercises/Array&Loops/arrayLoops.js

```diff
@@ -44,7 +45,6 @@ const determinatedOddEvenCards = (array, boolean) => {
     } else {
         return oddNumbers.length;
     }
-
```

Comment:
Both versions are correct, i don't know what the question below is refering to :)